src/qt/walletmodel.cpp

```diff
@@ -423,6 +428,11 @@ static void NotifyWatchonlyChanged(WalletModel *walletmodel, bool fHaveWatchonly
                               Q_ARG(bool, fHaveWatchonly));
 }
 
+static void NotifyCanGetAddressesChanged(WalletModel *walletmodel)
+{
+    QMetaObject::invokeMethod(walletmodel, "updateCanGetAddresses", Qt::QueuedConnection);
```

Comment:
I think it's safe to omit `Qt::QueuedConnection` - if current thread is not the same as the `wallet_model`'s thread then the default `Qt::AutoConnection` will do the same.

I think you can also call the signal `invokeMethod(wallet_model, "canGetAddressChanged")` and avoid the proxy slot `updateCanGetAddress`.

Reply:
Done. I've removed the proxy slot.